.github/workflows/page-build-and-deploy.yml

```
@@ -0,0 +1,33 @@
+  workflow_dispatch:
+
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+jobs:
+  build-and-deploy:
+    environment:
+      name: github-pages
+      url: ${{ steps.deployment.outputs.page_url }}
```

actionlint (1.7.12)

[error] property "deployment" is not defined in object type {}

(expression)